Lib/ntpath.py

```diff
@@ -525,14 +550,9 @@ def normpath(path):
             curdir = '.'
             pardir = '..'
         path = path.replace(altsep, sep)
-        prefix, path = splitdrive(path)
-
-        # collapse initial backslashes
-        if path.startswith(sep):
-            prefix += sep
-            path = path.lstrip(sep)
+        drive, root, path = splitroot(path)
 
-        comps = path.split(sep)
+        comps = path.lstrip(sep).split(sep)
```

Comment:
I'm surprised that this needs an additional `lstrip(sep)`, since that's exactly what `splitroot` should now be putting into `root`?

Reply:
The path isn't normalized yet. For example, say it's `"D:\\\\spam\\eggs"`, which splits as `('D:', '\\', '\\spam\\eggs')`.

Reply:
Ah right, before that was in the now-removed if-branch. 👍